pkg/virt-launcher/monitor.go

```diff
@@ -150,6 +151,13 @@ func (mon *monitor) refresh() {
 		return
 	}
 
+	if isZombie {
+		log.Log.Infof("Process %s and pid %d is a zombie, sending SIGCHLD to pid 1 to reap process", mon.cmdlineMatchStr, mon.pid)
+		syscall.Kill(1, syscall.SIGCHLD)
```

Comment:
is this still okay in the non-root case?

Reply:
It should be. Pid 1 is going to be virt-launcher regardless (both root and non-root have their own pid namespace), and this process calling syscall.Kill should have the same ownership as whatever pid 1 is to allow the signal to be sent

Reply:
yeah, perfect :)